components/infobox/commons/infobox_team.lua

```diff
@@ -206,7 +206,7 @@ end
 ---@return string|number|nil # storage date
 ---@return string[] # display elements
 function Team:processCreateDates()
-	local earliestGameTimestamp = Team._parseDate(self.args.created) or Date.maxTimestamp
+	local earliestGameTimestamp = Team._parseDate(ReferenceCleaner.clean(self.args.created)) or Date.maxTimestamp
```

Comment:
can we kick the `ReferenceCleaner.clean` call in `_setLpdbData` due to the changes?

for legacy input this reads the first date it comes across in the input string, right?